models/base_model.py

```diff
@@ -1311,84 +1258,47 @@ def forward_semantic_cls(self):
             if not self.opt.train_cls_regression:
                 _, self.pfB = self.pred_cls_fake_A.max(1)
 
-    def compute_metrics(self, n_epoch, n_iter):
-
+    def compute_metrics(self):
         dims = 2048
         batch = 1
 
-        # B->A
-        if hasattr(self, "netG_B"):
-
-            if len(self.fake_A_pool.get_all()) > 0:
-                (
-                    self.fakemA,
-                    self.fakesA,
-                    self.fakeactA,
-                ) = _compute_statistics_of_dataloader(
-                    path_sv=None,
-                    model=self.netFid,
-                    domain=None,
-                    batch_size=batch,
-                    dims=dims,
-                    dataloader=self.fake_A_pool.get_all(),
-                    device=self.gpu_ids[0],
-                    nb_max_img=self.opt.train_nb_img_max_fid,
-                    root=self.root,
-                )
-
-                self.fidA = calculate_frechet_distance(
-                    self.realmA, self.realsA, self.fakemA, self.fakesA
-                )
-
         # A->B
 
-        self.fakemB, self.fakesB, self.fakeactB = _compute_statistics_of_dataloader(
+        self.fakeactB = _compute_statistics_of_dataloader(
             path_sv=None,
             model=self.netFid,
             domain=None,
-            batch_size=batch,
+            batch_size=1,
```

Comment:
Above variable `batch`  instead ?
Side question is do we want `dims`and `batch` hardcoded ?